src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -80,23 +72,34 @@ const messageEditInput = 'messageEditInput';
 
 const isMobileSafari = Browser.isMobileSafari();
 
-function ReportActionItemMessageEdit(props) {
+function ReportActionItemMessageEdit({
+    action,
+    draftMessage,
+    reportID,
+    index,
+    shouldDisableEmojiPicker = false,
+    preferredSkinTone = CONST.EMOJI_DEFAULT_SKIN_TONE,
+    forwardedRef = () => {},
+}: ReportActionItemMessageEditProps & {
+    forwardedRef: ForwardedRef<TextInput & HTMLTextAreaElement>;
+}) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const reportScrollManager = useReportScrollManager();
     const {translate, preferredLocale} = useLocalize();
+    // @ts-expect-error TODO: Remove this once useKeyboardState (https://github.com/Expensify/App/issues/24941) is migrated to TypeScript.
```

Comment:
It's already migrated, no need for this comment

Reply:
@blazejkustra we have issue on useKeyboardState.
I need to fix that?

Reply:
What issue exactly?

Reply:
```js
const {isKeyboardShown} = useKeyboardState()
```
`isKeyboardShown` can be null. so I couldn't get this value without ts-ignore

Reply:
Let's fix useKeyboardState then (add a default value perhaps?)

Reply:
hi, @blazejkustra updated. please check.

Reply:
Hi, @blazejkustra Could you check?